Update motivation.md (#43)
sgibson91/cross-stitch-carpentry: _extras/motivation.md

title: Motivation
---

> # Cross Stitching is like computer programming

Reviewer: ```suggestion
# Cross Stitching is like Computer Programming
```

> Computer programming or coding requires analytical skills, which allows us to think of a solution to a problem, design a plan comprising of multiple steps and accomplish them by writing multiple lines of code. 

Reviewer: ```suggestion
Computer programming or coding requires analytical skills.
These skills allow us to think of a solution to a problem, design a plan comprising of multiple steps and accomplish them by writing multiple lines of code. 
```

Just breaking this sentence up a bit as it was quite long

> Each of these lines have been intentionally written to play a vital role in a working software.

Reviewer: ```suggestion
Each of these lines have been intentionally written to play a vital role in a software package or analytical pipeline.
```

> Cross stitching is quite like programming where you create a beautiful pattern by "writing" in rows. 

Reviewer: ```suggestion
Cross stitching is quite like programming where the input is the combination of colours, the algorithm is the pattern chart, and the output is a beautiful design you have created.

As you stitch, you have to decode the pattern symbols into thread colours and the pattern is often built up across rows.
```

> If there’s a bug or mis-stitching in a row, you have to fix that to create your pattern as you intented to look like.

Reviewer: ```suggestion
If there’s a bug or mis-stitching in a row, you have to fix that to create your pattern as you intended it to look like.
```

Keeping that in mind, we invite you to explore cross-stitching as a hobby for programmers. 

> This lesson is designed to train a novice learner to learn how to cross stitch and use it as a hobby to mindfully enjoy their breaks from work.

Reviewer: ```suggestion
This lesson is designed to train a beginner to learn how to cross stitch and use it as a hobby to mindfully enjoy their breaks from work.
```

Repetition of "learn"

# Why cross stitch as a programmer?

> On a daily basis, we use smartphone and computers for everything we do. 

Reviewer: ```suggestion
On a daily basis, we use smartphones and computers for everything we do. 
```

> Starting from getting a fast access to news and latest information in the morning, to learning a new programming language or other skills to getting our job done at work, to ordering things that makes our day-to-day life possible.

Reviewer: ```suggestion
Starting from getting instant access to news and the latest information in the morning, to learning a new programming language or other skills for getting our job done at work, to online shopping that makes our day-to-day life possible.
```

> Needless to say, such excessive use of technology leads to information overload, and often results into burnout and other mental health issues including anxiety, and depression [1].

Reviewer: ```suggestion
Needless to say, such excessive use of technology leads to information overload and often results in burnout and other mental health issues, such as anxiety and depression [1].
```

> <SARAH HELP ME>

Reviewer: ```suggestion
## Sarah's Impact Story

I was taught to cross stitch in my early teens by my aunt.
It was also the same time she got me into heavy metal music and took to me to my first rock music gigs.
She is still by far the coolest member of my family!

I stuck with cross stitching as a hobby for a number of reasons, but most of all I do it because it keeps my hands busy.
In an evening, I would often find myself mindlessly scrolling through social media when I was supposed to be paying attention to a film or a conversation with my friend.
Cross stitching keeps me off my phone when I don't need to be, but still leaves me with enough "brain space" to carry on a conversation or watch something.
(My new favourite thing to stitch along to is audiobooks!)

Another reason I have continued to cross stitch is that it forces me to slow down.
If I rush stitching, I'm bound to make a mistake.
So instead I take my time to make sure I have the right coloured thread and I've counted the correct number of spaces. _Count twice, stitch once!_
```

# Mindful Hobbies in Digital Age

> *Mindfulness and technology is a movement in research and design, that encourages the user to become aware of the present moment, rather than losing oneself in a technological device. This field encompasses multidisciplinary participation between design, psychology, computer science, and religion.* Source: [Wikipedia](https://en.wikipedia.org/wiki/Mindfulness_and_technology)

Mindfulness has been theoretically and empirically associated with psychological well-being [2]. 
Practicing mindfulness through a habby is an easy way to intentionally separate our 'down-time' from 'work-time' and boost our wellbeing. 

> It is also recommended by National Institute for Health and Care Excellence to boost attention and concentration, and to prevent recurrent depression [3].

Reviewer: ```suggestion
The [National Institute for Health and Care Excellence](INSERT LINK) recommends mindfulness as a way to boost attention and concentration and to prevent recurrent depression [3].
```

Cross stitching keeps me off my phone when I don't need to be, but still leaves me with enough "brain space" to carry on a conversation or watch something.
(My new favourite thing to stitch along to is audiobooks!)

Another reason I have continued to cross stitch is that it forces me to slow down.
If I rush stitching, I'm bound to make a mistake.
So instead I take my time to make sure I have the right coloured thread and I've counted the correct number of spaces. _Count twice, stitch once!_

# Mindful Hobbies in Digital Age

> *Mindfulness and technology is a movement in research and design, that encourages the user to become aware of the present moment, rather than losing oneself in a technological device. This field encompasses multidisciplinary participation between design, psychology, computer science, and religion.* Source: [Wikipedia](https://en.wikipedia.org/wiki/Mindfulness_and_technology)

Mindfulness has been theoretically and empirically associated with psychological well-being [2]. 
Practicing mindfulness through a habby is an easy way to intentionally separate our 'down-time' from 'work-time' and boost our wellbeing. 

> The [National Institute for Health and Care Excellence](INSERT LINK) recommends mindfulness as a way to boost attention and concentration and to prevent recurrent depression [3].

Author: ```suggestion
The [National Institute for Health and Care Excellence](https://www.nice.org.uk/) recommends mindfulness as a way to boost attention and concentration and to prevent recurrent depression [3].
```

[3] [Look after your mental health using mindfulness](https://www.mentalhealth.org.uk/sites/default/files/How%20to...mindfulness.pdf).
[4] [Why Knitting Is the Must-Have Life Skill](https://www.barbarahannahgrufferman.com/knit-your-way-to-cool-calm-and-healthy/), Barbara Hannah Grufferman, [featured on Huffpost]((https://www.huffpost.com/entry/learn-to-knit-_b_7146404?guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAEkxPdsb_i9Sh2ZeDeJbCYN-etQuQ9yk4C7D0zEm391g0aJsrvCWy8BI-UC1aNRLB7HOnle6QxFTr6zygm6M4C13SQKtC7N4D4RDVtaDruszwK8Wm-LsmRxERGCdtXDDzM7RMywhsQ69e8OY-dKT1W0VCI5mIjfGbcJ9Ch7BF-cu&guccounter=2) on 05 February 2014
[5] [Re-evaluating whether bilateral eye movements influence memory retrieval](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227790), Roberts BRT, Fernandes MA, MacLeod CM (2020) Re-evaluating whether bilateral eye movements influence memory retrieval. PLOS ONE 15(1): e0227790. https://doi.org/10.1371/journal.pone.0227790

> [6] [Social and leisure activities and risk of dementia: a prospective longitudinal study](https://www.ncbi.nlm.nih.gov/pubmed/7730528). PMID: 7730528, DOI: 10.1111/j.1532-5415.1995.tb06093.

Reviewer: ```suggestion
[1] [Extreme mediation: Observing mental and physical health in everyday life](http://ubicomp.org/ubicomp2014/proceedings/ubicomp_adjunct/posters/p47-faiola.pdf), UbiComp’14Adjunct, September 13-17, 2014, Seattle, WA, USAACM 978-1-4503-3047-3/14/09.http://dx.doi.org/10.1145/2638728.2638741

[2] [Effects of Mindfulness on Psychological Health: A Review of Empirical Studies](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679190/), Clin Psychol Rev. 2011 Aug; 31(6): 1041–1056. doi: 10.1016/j.cpr.2011.04.006.

[3] [Look after your mental health using mindfulness](https://www.mentalhealth.org.uk/sites/default/files/How%20to...mindfulness.pdf).

[4] [Why Knitting Is the Must-Have Life Skill](https://www.barbarahannahgrufferman.com/knit-your-way-to-cool-calm-and-healthy/), Barbara Hannah Grufferman, [featured on Huffpost]((https://www.huffpost.com/entry/learn-to-knit-_b_7146404?guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAEkxPdsb_i9Sh2ZeDeJbCYN-etQuQ9yk4C7D0zEm391g0aJsrvCWy8BI-UC1aNRLB7HOnle6QxFTr6zygm6M4C13SQKtC7N4D4RDVtaDruszwK8Wm-LsmRxERGCdtXDDzM7RMywhsQ69e8OY-dKT1W0VCI5mIjfGbcJ9Ch7BF-cu&guccounter=2) on 05 February 2014

[5] [Re-evaluating whether bilateral eye movements influence memory retrieval](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227790), Roberts BRT, Fernandes MA, MacLeod CM (2020) Re-evaluating whether bilateral eye movements influence memory retrieval. PLOS ONE 15(1): e0227790. https://doi.org/10.1371/journal.pone.0227790

[6] [Social and leisure activities and risk of dementia: a prospective longitudinal study](https://www.ncbi.nlm.nih.gov/pubmed/7730528). PMID: 7730528, DOI: 10.1111/j.1532-5415.1995.tb06093.
```

Reformatting so these appear on individual lines

## Reference

[1] [Extreme mediation: Observing mental and physical health in everyday life](http://ubicomp.org/ubicomp2014/proceedings/ubicomp_adjunct/posters/p47-faiola.pdf), UbiComp’14Adjunct, September 13-17, 2014, Seattle, WA, USAACM 978-1-4503-3047-3/14/09.http://dx.doi.org/10.1145/2638728.2638741

[2] [Effects of Mindfulness on Psychological Health: A Review of Empirical Studies](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679190/), Clin Psychol Rev. 2011 Aug; 31(6): 1041–1056. doi: 10.1016/j.cpr.2011.04.006.

[3] [Look after your mental health using mindfulness](https://www.mentalhealth.org.uk/sites/default/files/How%20to...mindfulness.pdf).

> [4] [Why Knitting Is the Must-Have Life Skill](https://www.barbarahannahgrufferman.com/knit-your-way-to-cool-calm-and-healthy/), Barbara Hannah Grufferman, [featured on Huffpost]((https://www.huffpost.com/entry/learn-to-knit-_b_7146404?guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAEkxPdsb_i9Sh2ZeDeJbCYN-etQuQ9yk4C7D0zEm391g0aJsrvCWy8BI-UC1aNRLB7HOnle6QxFTr6zygm6M4C13SQKtC7N4D4RDVtaDruszwK8Wm-LsmRxERGCdtXDDzM7RMywhsQ69e8OY-dKT1W0VCI5mIjfGbcJ9Ch7BF-cu&guccounter=2) on 05 February 2014

Author: ```suggestion
[4] [Why Knitting Is the Must-Have Life Skill](https://www.barbarahannahgrufferman.com/knit-your-way-to-cool-calm-and-healthy/), Barbara Hannah Grufferman, [featured on Huffpost](https://www.huffpost.com/entry/learn-to-knit-_b_7146404?guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAEkxPdsb_i9Sh2ZeDeJbCYN-etQuQ9yk4C7D0zEm391g0aJsrvCWy8BI-UC1aNRLB7HOnle6QxFTr6zygm6M4C13SQKtC7N4D4RDVtaDruszwK8Wm-LsmRxERGCdtXDDzM7RMywhsQ69e8OY-dKT1W0VCI5mIjfGbcJ9Ch7BF-cu&guccounter=2) on 05 February 2014
```
